fcrepo-fixity/src/test/java/org/fcrepo/camel/fixity/integration/RouteIT.java

```diff
@@ -100,20 +98,25 @@ protected Properties useOverridePropertiesWithPropertiesComponent() {
     @Override
     protected void addServicesOnStartup(final Map<String, KeyValueHolder<Object, Dictionary>> services) {
         final String jmsPort = System.getProperty("fcrepo.dynamic.jms.port", "61616");
+        final String webPort = System.getProperty("fcrepo.dynamic.test.port", "8080");
         final ActiveMQComponent component = new ActiveMQComponent();
 
         component.setBrokerURL("tcp://localhost:" + jmsPort);
         component.setExposeAllQueues(true);
 
-        services.put("broker", asService(component, "osgi.jndi.service.name", "fcrepoqueue"));
+        final FcrepoComponent fcrepo = new FcrepoComponent();
+        fcrepo.setBaseUrl("http://localhost:" + webPort + "/fcrepo/rest");
+
+        services.put("broker", asService(component, "osgi.jndi.service.name", "fcrepo/Broker"));
+        services.put("fcrepo", asService(fcrepo, "osgi.jndi.service.name", "fcrepo/Camel"));
     }
 
     @Test
     public void testFixityOnBinary() throws Exception {
         final String webPort = System.getProperty("fcrepo.dynamic.test.port", "8080");
         final String jmsPort = System.getProperty("fcrepo.dynamic.jms.port", "61616");
         final String path = fullPath.replaceFirst("http://localhost:[0-9]+/fcrepo/rest", "");
-        final String fcrepoEndpoint = "mock:fcrepo:localhost:" + webPort + "/fcrepo/rest";
+        final String fcrepoEndpoint = "mock:fcrepo:http://localhost:8080/fcrepo/rest";
```

Comment:
Because the URI for the camel endpoint comes from the default blueprint configuration (i.e. http://localhost:8080/fcrepo/rest), but we're overriding that on a component level (the whole rationale behind this PR) so it's no longer necessary to override the property value on the endpoint in this test (i.e. https://github.com/fcrepo4-exts/fcrepo-camel-toolbox/pull/101/files#diff-75a045e622ebabf3b4ce98360bf63f69L93). Thus, the endpoint (at runtime) is still `fcrepo:http://localhost:8080/fcrepo/rest` even though the endpoint reaches out to the location defined on a component level. Does that make any sense?